### PR TITLE
Create BNFabb120.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record.

### CODE REVIEW

File: ParisBNF/abb/BNFabb120.xml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                     
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="32r"/>
+                        <title type="complete" ref="LIT2505Weddas#Wednesday">Prayer for Wednesday attributed to Ephrem the Syrian</title>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.4">
+                        <locus from="47r"/>
+                        <title type="complete" ref="LIT2505Weddas#Thursday">Prayer for Thursday attributed to John of Saba
+                           (described by C. Conti Rossini as an excerpt from the work ʾAragāwi manfasāwi)</title>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml:59]: The information in brackets may look better in a note.

File: ParisBNF/abb/BNFabb120.xml [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf" quantity="195">195</measure>
+                           <measure unit="leaf" type="blank">4<locus target="#13r #46v #66v #104r"/></measure>
+                           <note>According to A.dʼAbbadie, six leaves are blank</note>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml:93]: Add space between "A.dʼAbbadie".

File: ParisBNF/abb/BNFabb120.xml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+                           <dimensions type="outer">
+                              <height unit="mm" cert="medium">340</height>
+                              <width unit="mm" cert="medium">300</width>
+                           </dimensions>
+                        </extent>
+                        <condition key="good">The spine is slightly damaged</condition>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml:99]: End with period.

File: ParisBNF/abb/BNFabb120.xml [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+                              <material key="textile"/>
+                           </decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate notBefore="1700" notAfter="1799" source="lettering">18th century</origDate>.                     
+                </origin>
+                  <provenance>The name of the original owner (or copyst) was erased and replaced with the name
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml:156]: copyist

File: ParisBNF/abb/BNFabb120.xml [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+                           </decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate notBefore="1700" notAfter="1799" source="lettering">18th century</origDate>.                     
+                </origin>
+                  <provenance>The name of the original owner (or copyst) was erased and replaced with the name 
+                     <persName ref="PRS14561WaldaTaklaHaymanot" role="owner">Wālda Takla Hāymānot</persName>.
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml:157]: Walda or Wālda?

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

File: ParisBNF/abb/BNFabb120.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb120.xml:158]: done

File: ParisBNF/abb/BNFabb120.xml [hunk 6]
```diff
@@ -90,7 +91,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <extent>
                            <measure unit="leaf" quantity="195">195</measure>
                            <measure unit="leaf" type="blank">4<locus target="#13r #46v #66v #104r"/></measure>
-                           <note>According to A.dʼAbbadie, six leaves are blank</note>
+                           <note>According to A.dʼAbbadie, six leaves are blank.</note>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb120.xml]: space in A.d'Abbadie -> A. d'Abbadie

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: ParisBNF/abb/BNFabb120.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb120.xml]: done

**Pull request merged**